modules/common/labels/labels.go

```diff
@@ -60,3 +61,28 @@ func GetLabels(
 
 	return util.MergeStringMaps(labelSelector, custom)
 }
+
+// GetSingleLabelSelector - create a simple label selector for a single key/value label
+func GetSingleLabelSelector(
+	key string,
+	value string,
+) metav1.LabelSelector {
+	return metav1.LabelSelector{
+		MatchExpressions: []metav1.LabelSelectorRequirement{
+			{
+				Key:      key,
+				Operator: metav1.LabelSelectorOpIn,
+				Values: []string{
+					value,
+				},
+			},
+		},
+	}
+}
+
+// GetAppLabelSelector - create a simple label selector for the default AppSelector "service" label
```

Comment:
@olliewalsh I don't fully understand this: why we can't simply call `GetSingleLabelSelector(common.AppSelector, value)` from an operator? `common.AppSelector` is something imported everywhere, and already available when building labels. I think we don't need `GetAppSelector` as wrapper for a function where one of the parameters is a const available in the service operators.

Reply:
Aren't both just convenience functions?
Almost all consumers can just call `GetAppLabelSelector(instance.name)`, special cases that don't use the service label can use `GetSingleLabelSelector("foo", "bar")`

Reply:
ack sounds good to have something like this